packages/telemetry/src/index.ts

```diff
@@ -30,6 +30,7 @@ export interface TelemetryApi {
   findActiveSpan(fn: findActiveSpanOptions): Span | undefined
   endActiveSpanAndChildren (span?: Span | undefined): void
   getActiveContextObject (): contextObject
+  getResources (): {}
```

Comment:
is it always a key value pair of strings in this case kinda similar to the tests?
```suggestion
  getResources (): `{[key: string]: string}`
```

Reply:
I know it's always serializable, but it might be numbers too

Reply:
maybe `getResources (): {[key: string]: string | number}`?

Reply:
lol, i tried that it's more complex (how about booleans? how about arrays of strings?). I'm going to try to reuse an opentelmetry type.

Reply:
worse case I feel like `{[key: string]: string}` is acceptable since it sounds like it could be any serializable type. Or it is always a primitive we could do something like `{[key: string]: string | number | boolean | undefined | null}` (assuming no symbols or bigint)?

Reply:
The otel type appears to be working for us.